jobs/config.json

```diff
@@ -4651,6 +4651,28 @@
       "sig-network"
     ]
   },
+  "ci-kubernetes-e2e-gci-gce-ipvs": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gci-gce-ipvs.env",
```

Comment:
You should just be able to set the environment variable directly rather than referencing a file

Reply:
true, wanted to follow the convention used in the other jobs that specified multiple files instead of the environment variable directly.

Reply:
Ah ok, then go ahead and stick with the convention

Reply:
if you can reuse an existing env file, please reuse. If the env is for your job specifically, you can use `--env=var=val` flag instead. We want to retire the .env files in the near future.

Reply:
done